libs/features/placeholders.js

```diff
@@ -1,20 +1,20 @@
+import { customFetch, getConfig } from '../utils/utils.js';
```

Comment:
As a pattern, we tend to avoid circular dependencies like these. Are we fine to keep them here?

Reply:
We had a circular dependency before already.
`Utils.js -> placeholders.js -> helpers.js (customFetch) -> utils.js`

I remember we discussed with @auniverseaway a while back that the only reason we avoided them was linting rules, which we changed.

I haven't seen it leading to any issues yet 🤷